vignettes/baselinenowcast.Rmd

```diff
@@ -216,40 +203,42 @@ target_data <- enw_filter_reference_dates(
   latest_data,
   include_days = n_training_volume - 1
 )
+```
 
-# Get the reporting triangle, adding an additional day because epinowcast
-# we want the max_delay + 1 entries since 0 is a valid delay.
-# This also validates that the data is in the correct format and
-# runs preprocessing see ?enw_preprocess_data for more details
+Next we obtain the reporting triangle, adding an additional day to the
+`max_delay` because we want the `max_delay + 1` entries since 0 is a valid
+delay.This also validates that the data is in the correct format and
+runs preprocessing see ?enw_preprocess_data for more details.
```

Comment:
should have code breaks about the ?enw_preprocess_data

Reply:
added text between each function to explain what data it is making

Reply:
hmm that isn't what I meant. What I meant was this should have `` around it or better yet use `function()` like all the others?